AnkiHubSoftware/ankihub_addon
# [BUILD-297] Remove the configure deleted notes dialog

## Related issues

https://ankihub.atlassian.net/browse/BUILD-297

## Proposed changes

This removes the "Configure deleted notes" dialog shown when installing a deck and set `BehaviorOnRemoteNoteDeleted.DELETE_IF_NO_REVIEWS` as the default value.


## How to reproduce

1. Subscribe and install a new deck.
2. Notice the configure deleted notes dialog is no longer shown.
3. Go to Deck Management to confirm that the "Remove notes without review history locally when deleted on AnkiHub" option is set.
